input/fsh/profiles/EncounterBirth.fsh

```diff
@@ -32,7 +32,7 @@ Description: "This Encounter profile represents the child's birth encounter."
   * individual 1.. MS
   * individual only Reference(PractitionerVitalRecords)
 * hospitalization MS
-  * destination only Reference(USCoreLocation)
+  * destination only Reference(LocationBFDR)
   * destination MS
     * ^short = "If infant transferred, facility transferred to"
     * ^definition = "If infant transferred, facility transferred to"
```

Comment:
"HOSPTO" is mapped to hospitalization.destination.name - we should have a short description in encounter birth so it shows in the differential

Reply:
Maps to LocationBFDR